bin/desimodel_sync_kpno.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# This script is intended to be run from a cronjob (see 
+# desimodel/etc/desimodel_sync_kpno_cron.sh).  You should not run
+# this script interactively.  It expects to be run inside an
+# already-configured software environment.
+
+fpsync=$(which desi_sync_focalplane)
+echo "Using focalplane sync script:  ${fpsync}"
+
+# Find the newest calibration file
+caldir="/data/focalplane/calibration/"
+calfile=$(ls ${caldir} | egrep '[0-9]{8}T[0-9]{6}.*' | sort | tail -n 1)
+calpath="${caldir}/${calfile}"
+
+# Run it.
```

Comment:
Should we do an "svn update $DESIMODEL/data" prior to running the sync?  In normal operations this cronjob would be the only thing updating the focal plane files, but there is the possibility that another update could come via a different route that would cause an svn error at the time of commit.

Reply:
The svn update is already done directly in `desi_sync_focalplane`.